website/docs/r/ssm_maintenance_window_target.html.markdown

```diff
@@ -73,3 +73,11 @@ The following arguments are supported:
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The ID of the maintenance window target.
+
+## Import
+
+SSM Maintenance Window targets can be imported using `WINDOW_TARGET_ID:WINDOW_ID`, e.g.
+
+```
+$ terraform import aws_ssm_maintenance_window_target.example 23639a0b-ddbc-4bca-9e72-78d96EXAMPLE:mw-0c50858d01EXAMPLE
```

Comment:
Needs to be updated to the new format: 😄 

``````suggestion
SSM Maintenance Window targets can be imported using `WINDOW_ID/WINDOW_TARGET_ID`, e.g.

```
$ terraform import aws_ssm_maintenance_window_target.example mw-0c50858d01EXAMPLE/23639a0b-ddbc-4bca-9e72-78d96EXAMPLE
``````

Reply:
ahh totally missed this file! 🙏